### DISCUSSION TITLE
Using etcd on separate dedicated nodes

### DISCUSSION THREAD

**Discussion by vasilytray** [autobase-tech/autobase#531] [General]
Hello, there is such a question as how to configure the configuration so that etcd can be placed on separate nodes allocated for this? And will the DCS cluster be able to serve multiple Postgres clusters?

**Comment by vitabaks** [comment]
Hello.

It is recommended to deploy the DCS cluster on dedicated servers, separate from the database servers. \
A dedicated DCS cluster can be used by multiple Patroni clusters.

Just specify the IP addresses (dedicated nodes) in the inventory file in the "etcd_cluster" group.

**Reply by vasilytray** [reply]
Thank you! that is, should I deploy the etcd cluster on separate nodes and specify the IP of these nodes in the inventory file configuration? I understand correctly?

**Reply by vitabaks** [reply]
If you want our automation to deploy an ETCD cluster, then specify the IP addresses in the inventory. 

If you want to manage the ETCD cluster yourself, then leave the "etcd_cluster" group empty, and specify the variable 'dcs_exists: true' and patroni_etcd_hosts in vars/main.yml

**Reply by vasilytray** [reply]
Thank you!

**Reply by vasilytray** [reply]
If i want to manage the ETCD cluster myrself, after changing the etcd settings, will the logic of haproxy balancers change?

**Reply by vitabaks** [reply]
No.
